partiql-app/partiql-cli/src/main/kotlin/org/partiql/cli/pico/PipelineOptions.kt

```diff
@@ -22,28 +23,27 @@ import org.partiql.lang.eval.ProjectionIterationBehavior
 import org.partiql.lang.eval.TypedOpBehavior
 import org.partiql.lang.eval.TypingMode
 import org.partiql.lang.eval.UndefinedVariableBehavior
-import org.partiql.pipeline.AbstractPipeline
 import picocli.CommandLine
 import java.io.File
 
 internal class PipelineOptions {
 
-    @CommandLine.Option(names = ["--pipeline"], description = ["The type of pipeline to use"], paramLabel = "TYPE")
+    @CommandLine.Option(names = ["-p", "--pipeline"], description = ["The type of pipeline to use: [\${COMPLETION-CANDIDATES}]"], paramLabel = "TYPE")
     var pipelineType: AbstractPipeline.PipelineType = AbstractPipeline.PipelineType.STANDARD
 
     @CommandLine.Option(names = ["-e", "--environment"], description = ["File containing the global environment"], paramLabel = "FILE")
     var environmentFile: File? = null
 
-    @CommandLine.Option(names = ["--typing-mode"], description = ["Specifies the typing mode"], paramLabel = "MODE")
+    @CommandLine.Option(names = ["-t", "--typing-mode"], description = ["Specifies the typing mode: [\${COMPLETION-CANDIDATES}]"], paramLabel = "MODE")
     var typingMode: TypingMode = TypingMode.LEGACY
 
-    @CommandLine.Option(names = ["-t", "--typed-op-behavior"], description = ["Indicates how CAST should behave"], paramLabel = "BEHAVIOR")
+    @CommandLine.Option(names = ["-T", "--typed-behavior"], description = ["Indicates how CAST should behave: [\${COMPLETION-CANDIDATES}]"], paramLabel = "OPT")
     var typedOpBehavior: TypedOpBehavior = TypedOpBehavior.HONOR_PARAMETERS
 
-    @CommandLine.Option(names = ["-r", "--projection-iter-behavior"], description = ["Controls the behavior of ExprValue.iterator in the projection result"], paramLabel = "BEHAVIOR")
+    @CommandLine.Option(names = ["-I", "--iter-behavior"], description = ["Controls the behavior of ExprValue.iterator in the projection result: [\${COMPLETION-CANDIDATES}]"], paramLabel = "OPT")
     var projIterBehavior: ProjectionIterationBehavior = ProjectionIterationBehavior.FILTER_MISSING
 
-    @CommandLine.Option(names = ["-v", "--undefined-variable-behavior"], description = ["Defines the behavior when a non-existent variable is referenced"], paramLabel = "BEHAVIOR")
+    @CommandLine.Option(names = ["-u", "--undef-behavior"], description = ["Defines the behavior when a non-existent variable is referenced: [\${COMPLETION-CANDIDATES}]"], paramLabel = "OPT")
```

Comment:
Is there any particular reason to change the names/aliases from before? I like the consistency of using the single character as the first name alias. I would remove the `-I` altogether though as that's a loaded term/alias. I think you should keep `--undefined-variable-behavior` even though it's verbose because we have a good alias for it now. I'm inclined to remove the aliases from `--typing-mode` and `--typed-op-behavior`

Reply:
ACK. I renamed to use the longer terms and removed ambiguous aliases.